src/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -167,6 +167,7 @@
 	private static final String PARAM_IDX = "idx";
 	private static final String PARAM_IS_REGEX = "isRegex";
 	private static final String PARAM_IS_ENABLED = "isEnabled";
+	private static final String PARAM_RISK = "risk";
```

Comment:
`riskID`? Otherwise we should mention in the description that's expected the ID (not the name).